src/applications/vaos/tests/appointment-list/components/CommunityCareAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -428,53 +429,39 @@ describe('VAOS <CommunityCareAppointmentDetailsPage>', () => {
         .getAttribute('href')
         .replace('data:text/calendar;charset=utf-8,', ''),
     );
-    const tokens = ics.split('\r\n');
+    const tokens = getICSTokens(ics);
 
-    expect(tokens[0]).to.equal('BEGIN:VCALENDAR');
-    expect(tokens[1]).to.equal('VERSION:2.0');
-    expect(tokens[2]).to.equal('PRODID:VA');
-    expect(tokens[3]).to.equal('BEGIN:VEVENT');
-    expect(tokens[4]).to.contain('UID:');
+    expect(tokens.get('BEGIN').includes('VCALENDAR')).to.be.true;
+    expect(tokens.get('VERSION')).to.equal('2.0');
+    expect(tokens.get('PRODID')).to.equal('VA');
+    expect(tokens.get('BEGIN').includes('VEVENT')).to.be.true;
+    expect(tokens.has('UID')).to.be.true;
 
     // TODO: Should this be provider practice instead of name???
-    expect(tokens[5]).to.equal('SUMMARY:Appointment at Rick Katz');
+    expect(tokens.get('SUMMARY')).to.equal('Appointment at Rick Katz');
 
     // The description text longer than 74 characters should start newlines with a tab character
-    expect(tokens[6]).to.equal(
-      'DESCRIPTION:You have a health care appointment with a community care provi',
+    expect(tokens.get('DESCRIPTION')).to.equal(
+      'You have a health care appointment with a community care provi',
     );
-    expect(tokens[7]).to.equal(
-      '\tder. Please don’t go to your local VA health facility.',
+    expect(tokens.get('FORMATTED_TEXT')).to.equal(
+      '\tder. Please don’t go to your local VA health facility.\t\\n\\n123\\n\tBurke\\, VA 20151\\n\t(703) 555-1264\\n\t\\nSign in to https://va.gov/health-care/schedule-view-va-appointments/appo\tintments to get details about this appointment\\n',
```

Comment:
Is `FORMATTED_TEXT` something we create when parsing the tokens? It's confusing to me when reading the test why it starts from the middle of a work in the `DESCRIPTION` field.

Is it possible to put all this in the `DESCRIPTION` token value? And I wonder if splitting the text into an array of lines would make it clearer that we're testing the formatting correctly in the test.